README.md

```diff
@@ -34,46 +36,68 @@ aimed at react developers that want to embed the widgets into their applications
 npm install --save webex @webex/widgets
 ```
 
+Notice that the [Webex Javascript SDK](https://www.npmjs.com/package/webex)
+has be installed as a peer dependency.
+
 ## Webex Components vs Webex Widgets
 
-[Webex Components](https://github.com/webex/components#webex-components) is a set of React components that follow webex standard styling without any data being injected to them.
-[Webex Widgets](https://github.com/webex/widgets#webex-widgets) are based on [Webex components](https://github.com/webex/components#webex-components) but include the adapter that talks to our [Javascript SDK](https://github.com/webex/webex-js-sdk).
-This means that the Webex Widgets use the [SDK Component Adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter) to inject the [Webex data](https://github.com/webex/webex-js-sdk#webex-js-sdk) for you.
+In addition to the Webex Widgets, we also offer the
+[Webex Component System](https://github.com/webex/components#webex-components).
+The Webex Component System (sometimes shorten as _Webex Components_) is a set of
+React components that, while following Webex styling, allow for more granularity
+in terms of layout and source of data.
+To learn more on the Webex Component System head to its Github repository at
+https://github.com/webex/components.
+
+Webex Widgets are based on Webex Components but include the adapter that uses
+our [Javascript SDK](https://github.com/webex/webex-js-sdk) to talk to Webex services for you.
+This means that the Webex Widgets use the
+[SDK Component Adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter)
+to inject the Webex data.
+All you need is a valid access token and a few parameters based on the widget you want to use.
+
 You have to take the Widget layout as-is, but the benefit is that there are no configurations needed.
 Install, copy-paste and you have the power of Webex in your application!
 
 ## Usage
 
 ### Styles
 
-In order to properly style Webex Widgets, we need to import all the fonts, icons, images and core CSS manually.
+In order to properly style Webex Widgets, you will need to import our CSS separately.
 Import `@webex/widgets/dist/webexWidgets.css` into your main entry file.
 
 There are two ways to do this:
 
 #### JavaScript
 
-In your `App.js`, add the following import:
+In your `App.js(x)`, add the following import:
```

Comment:
What is this addition?

Reply:
This is supposed to be `App.js` or `App.jsx`, if it doesn't make sense I can revert it